### DISCUSSION TITLE
allure-pytest and some other plugins in the documentation annotated plugins list

### DISCUSSION THREAD

**Discussion by skhomuti** [pytest-dev/pytest#10284] [Ideas] [answered]
Hey pytest team! 

Would you mind if [allure-pytest](https://pypi.org/project/allure-pytest/) is added to the annotated plugins list in https://docs.pytest.org/en/7.1.x/how-to/plugins.html? 
On my view it's a really popular plugin that might be in this list.

Probably not only allure but some other plugins from community might be mentioned in this page. Yes, I know we have this list https://docs.pytest.org/en/7.1.x/reference/plugin_list.html#plugin-list with every pytest plugin but it looks a little wild and not very useful. At the same time, a curated list can be a great idea.

**Comment by nicoddemus** [comment, marked as answer]
Hi @skhomuti,

Sure, feel free to open a PR. 👍
